src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,6 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     final String question = "question";
     final String option = "option";
     final String questionValue = request.getParameter(question);
+    String [] questionValueIndex = questionValue.split(" ",0);
```

Comment:
please add a comment here.

Reply:
why are we using 0 heere?

Reply:
maybe add a validation check before split?

Reply:
I added to 0 because you never know how long the string of the question will be like and with 0 I can assure longer questions will be also split. 
I'm not sure what kind of validation are you saying. Can you give me an example for cases for validate?